lib/membrane_h264_ffmpeg/parser.ex

```diff
@@ -252,7 +252,11 @@ defmodule Membrane.H264.FFmpeg.Parser do
       Enum.concat([[state.frame_prefix || <<>>], sps, pps])
       |> Enum.join(<<0, 0, 1>>)
 
-    {:ok, %{state | frame_prefix: frame_prefix}}
+    if state.skip_until_parameters? do
+      Membrane.Logger.warn("Flag skip_until_parameters? is not compatible with H264.RemoteStream")
```

Comment:
I think we should be a little more verbose about the action that we are taking.
```suggestion
      Membrane.Logger.warn("""
      Flag skip_until_parameters? is not compatible with Membrane.H264.RemoteStream caps.
      It is being automatically disabled.
      """)
```